packages/@vuepress/theme-default/src/client/styles/transitions.scss

```diff
@@ -1,9 +1,9 @@
 .fade-slide-y-enter-active {
-  transition: all 0.3s ease;
+  transition: all var(--t-duration-slide) ease;
 }
 
 .fade-slide-y-leave-active {
-  transition: all 0.3s cubic-bezier(1, 0.5, 0.8, 1);
+  transition: all var(--t-duration-slide) cubic-bezier(1, 0.5, 0.8, 1);
```

Comment:
As the other parts of are not customizable, I think we could just leave the duration constant?

Reply:
9c8bf00